test/functional/test_framework/wallet.py

```diff
@@ -182,28 +182,30 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
-def make_chain(node, address, privkeys, parent_txid, parent_value, n=0, parent_locking_script=None, fee=DEFAULT_FEE):
-    """Build a transaction that spends parent_txid.vout[n] and produces one output with
+def make_chain(node, address, privkeys, parent_txid, parent_value, parent_locking_script=None, fee=DEFAULT_FEE):
+    """Build a transaction that spends parent_txid.vout[0] and produces one output with
     amount = parent_value with a fee deducted.
     Return tuple (CTransaction object, raw hex, nValue, scriptPubKey of the output created).
     """
-    inputs = [{"txid": parent_txid, "vout": n}]
+    inputs = [{"txid": parent_txid, "vout": 0}]
     my_value = parent_value - fee
-    outputs = {address : my_value}
+    outputs = {address: my_value}
     rawtx = node.createrawtransaction(inputs, outputs)
     prevtxs = [{
         "txid": parent_txid,
-        "vout": n,
+        "vout": 0,
         "scriptPubKey": parent_locking_script,
         "amount": parent_value,
     }] if parent_locking_script else None
     signedtx = node.signrawtransactionwithkey(hexstring=rawtx, privkeys=privkeys, prevtxs=prevtxs)
     assert signedtx["complete"]
     tx = tx_from_hex(signedtx["hex"])
-    return (tx, signedtx["hex"], my_value, tx.vout[0].scriptPubKey.hex())
+    return tx, signedtx["hex"], my_value, tx.vout[0].scriptPubKey.hex()
```

Comment:
I think https://github.com/bitcoin/bitcoin/pull/21800#discussion_r685061119 suggested changing this return value to a named tuple instead?